docker/rucio_client/scripts/setWmcoreTapeOutput

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python3
+
+"""
+DM whitelists tape sites via wmcore_output_tape RSE attribute for Production Output. 
```

Comment:
I would prefer to get rid of whitelist and blacklist everywhere we possibly can. Some for of allowed or allowed lists should suffice along with banned lists.

Reply:
Very good point, changed it